spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -978,4 +978,5 @@ public ZeppelinContext getZeppelinContext() {
   public SparkVersion getSparkVersion() {
     return sparkVersion;
   }
+
```

Comment:
I had introduced a new function to the SparkInterpreter, but then deleted it, accidently left a blank line in. I've deleted this change.